qemu/tests/cfg/plug_cdrom.cfg

```diff
@@ -0,0 +1,41 @@
+- plug_cdrom:
+    only virtio_scsi
+    virt_test_type = qemu
+    type = plug_cdrom
+    cdroms = 'cd2'
+    iso_name_cd2 = new
+    cdrom_cd2 = /tmp/${iso_name_cd2}.iso
+    pre_command_noncritical = no
+    pre_command = 'dd if=/dev/urandom of=/tmp/${iso_name_cd2} bs=10M count=1 &&'
+    pre_command += ' mkisofs -o /tmp/${iso_name_cd2}.iso /tmp/${iso_name_cd2}'
+    post_command = "rm -rf /tmp/${iso_name_cd2}.iso /tmp/${iso_name_cd2}"
+    kill_vm_on_error = yes
+    driver_name = vioscsi
+    items_checked_cd2 = '{"io-status": "ok", "device": "drive_${cdroms}", "removable": True, '
+    items_checked_cd2 += '"file": "/tmp/${iso_name_cd2}.iso"}'
+    variants:
+        - shutdown_vm:
+            shutdown_vm = yes
+            command_qmp = system_powerdown
+            Windows:
+                command_shell = shutdown -t 0 -s
+            Linux:
+                command_shell = shutdown -h now
+        - reboot_vm:
```

Comment:
In my understanding, this scenario is reboot then shutdown, right?

Reply:
Nope, It' s two scenario, one is reboot, another is shutdown.

Reply:
@qingwangrh, my mistakes, have updated it. pls help me to review it again, thanks.